subprojects/gst-plugins-bad/sys/msdk/gstmsdkvpp.c

```diff
@@ -599,11 +622,13 @@ gst_msdkvpp_decide_allocation (GstBaseTransform * trans, GstQuery * query)
     GST_ERROR_OBJECT (thiz, "Failed to get video info");
     return FALSE;
   }
-  /* if downstream allocation query supports dmabuf-capsfeatures,
-   * we do allocate dmabuf backed memory */
+  /* We allocate the memory of type that downstream allocation requests */
   if (_gst_caps_has_feature (caps, GST_CAPS_FEATURE_MEMORY_DMABUF)) {
     GST_INFO_OBJECT (thiz, "MSDK VPP srcpad uses DMABuf memory");
     thiz->use_srcpad_dmabuf = TRUE;
+  } else if (_gst_caps_has_feature (caps, GST_CAPS_FEATURE_MEMORY_VA)) {
```

Comment:
The GST_CAPS_FEATURE_MEMORY_VA should be protected by macros.

Reply:
Fixed.